mods/ts/rules/nod-vehicles.yaml

```diff
@@ -293,12 +293,21 @@ SAPC:
 		TurnSpeed: 5
 		Speed: 71
 		RequiresCondition: !empdisable && !loading
+		Subterranean: true
+		SubterraneanCondition: submerged
+		SubterraneanTransitionTerrainTypes: Clear
```

Comment:
You need to add `Rough` here, subterranean units were able to dig and emerge on the `Rough` terrain type in TS.